spec/requests/internal/moderator_logs_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "/internal/tags", type: :request do
+  let(:super_admin) { create(:user, :super_admin) }
+  let(:tag_moderator) { create(:user) }
+  let(:tag) { create(:tag) }
+  let(:listener) { :internal }
+
+  before do
+    tag
+    sign_in super_admin
+    Audit::Subscribe.listen listener
+  end
+
+  after do
+    Audit::Subscribe.forget listener
+  end
+
+  describe "POST /internal/tag/:id" do
+    [
```

Comment:
I would separate these cases (add, remove, update) for readability, but I'll leave this decision up to you.